koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py

```diff
@@ -32,9 +36,50 @@
 LOG = logging.getLogger(__name__)
 
 
-class OCPCloudParquetReportSummaryUpdater(OCPCloudReportSummaryUpdater):
+class OCPCloudParquetReportSummaryUpdater(PartitionHandlerMixin, OCPCloudUpdaterBase):
     """Class to update OCP report summary data."""
 
+    def get_infra_map(self, start_date, end_date):
```

Comment:
These 2 methods were copied over from `OCPCloudReportSummaryUpdater` and I've deleted `OCPCloudReportSummaryUpdater`.